core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
No work is getting delegated to the engine by this class simply because it extends Game.  Look at the source for the Game class.  Its render() method only checks if the current screen is null, then delegates to the current screen's render() method.

Inheriting from Game does nothing unless you pass it to an initialization method (like we're doing with FilePlayMain in the launchers).

The engine still controls when things happen.  The engine essentially calls FilePlayMain's (our initialized ApplicationAdapter) render() method, which then delegates to the screens, which delegate to other methods.